### PR TITLE
Javascript javascript2 week1

### PR DESCRIPTION
**Pull request by 27ManishaPatel**

### CODE REVIEW

**Review by Divyajg — APPROVED** [review]
Good coding. Keep it up
Only one suggestion for now, try to avoid in-styling as it will benefit you more in future.

**Comment by dpfernandes** [comment]
@andrii1 can you please review this PR

File: javascript/javascript2/week1/script.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+const danishString2 = "Blå grød med røde bær"
+
+function findDanLetter(x) {
+    const para = document.createElement("p");
+    para.innerText = x
+    myDivExe2.appendChild(para);
+
+    const letterÅ = /å/i;
+    const letterÆ = /æ/i;
+    const letterØ = /ø/i;
```
Comment (andrii1) [inline javascript/javascript2/week1/script.js]: Will these danish characters work ok? Maybe it is better to name variables like letterAa or letterAe

File: javascript/javascript2/week1/script.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    const foundÅ = (x.match(letterÅ) || []).length;
+    const foundÆ = (x.match(letterÆ) || []).length;
+    const foundØ = (x.match(letterØ) || []).length;
+    let total = foundÅ + foundÆ + foundØ;
+    let result = {
+        Total: total,
+        Å: foundÅ,
```
Comment (andrii1) [inline javascript/javascript2/week1/script.js]: Cool! You can also add few if statements so if there are no letters found - they are not in return e.g. returns {total: 1, å: 1}

File: javascript/javascript2/week1/script.js (hunk 1 above)
Comment (27ManishaPatel) [reply javascript/javascript2/week1/script.js]: yes as of now its working fine but you are right its not good idea to use danish letter as variables

File: javascript/javascript2/week1/script.js (hunk 2 above)
Comment (27ManishaPatel) [reply javascript/javascript2/week1/script.js]: true that is good point

**Pull request merged**